tests/fcma/test_mvpa_voxel_selection.py

```diff
@@ -43,6 +43,8 @@ def test_mvpa_voxel_selection():
                 output.append(int(8*tuple[1]))
         expected_output = [6, 6, 5, 5, 5, 5, 5, 5, 5, 4, 4, 4, 4, 4,
                            4, 4, 4, 3, 3, 3, 3, 3, 2, 2, 2, 1]
+
+        # Check that the mean accuracy with noise is close to chance
```

Comment:
@CameronTEllis, you missed a line when you undid the test change. Next time, try `git revert`.